src/Lean/Server/Completion.lean

```diff
@@ -61,27 +63,28 @@ private def isTypeApplicable (type : Expr) (expectedType? : Option Expr) : MetaM
   catch _ =>
     return false
 
-private def sortCompletionItems (items : Array CompletionItem) : Array CompletionItem :=
-  items.qsort fun i1 i2 => i1.label < i2.label
+private def sortCompletionItems (items : Array (CompletionItem × Float)) : Array CompletionItem :=
+  -- TODO fix float eq comparison
```

Comment:
Is there actually a problem here in practice? Since the denominator of the float division is the same for all items, the floats should (basically) be equal iff if the `Nat` scores were equal, no?

Reply:
I guess so, but this would make `sortCompletionItems` dependent on `fuzzyMatchScore?` (undocumented), instead of allowing arbitrary scores for `CompletionItem`s.

Is there a pattern for epsilon quality of `Float`s in Lean?

Reply:
Not directly, we don't even have `Float.abs` yet afaik. But I really think it's fine as is for now. In practice, sorting by label probably doesn't matter much for any score < 1.0.

Reply:
Alright :+1: